components/rich_text_editor/rich_text_editor.vue

```diff
@@ -147,6 +156,13 @@ export default {
       if (this.link) {
         extensions.push(this.getExtension(Link, this.link));
       }
+      // Enable placeholderText
+      extensions.push(
+        this.getExtension(
+          Placeholder,
+          { placeholder: this.placeholderText },
+        ),
+      );
```

Comment:
`this.getExtension` was a helper to handle both booleans and options passed for the extensions, but since we always want to pass options to this one we don't need to use that
```
extensions.push(
  Placeholder.configure({ placeholder: this.placeholder }),
);
```

Reply:
ohk, i just thought its a wrapper method made for this purpose. changed